metrics-api/src/main/java/software/amazon/swage/metrics/record/MetricRecorder.java

```diff
@@ -133,11 +137,12 @@ public void record(Metric label, Number value, Unit unit, Instant time) {
 
             @Override
             public void count(Metric label, long delta) {
-                MetricRecorder.this.count(label, delta, context);
+                counts.compute(label, (x, previous) -> (previous == null) ? delta : previous + delta);
```

Comment:
So calling count now no longer actually records a metric, but stores and aggregates (regardless of implementation).  Not sure this is good.

It should be up to the recorder implementation do decide if it wants to aggregate things before sending.